machine-learning/02-Data Preparation and Feature Engineering.ipynb

```diff
@@ -1,45 +1,405 @@
+        "    public float Trip_time_in_secs { get; set; }\n",
+        "\n",
+        "    [LoadColumn(4)]\n",
+        "    [ColumnName(@\"trip_distance\")]\n",
+        "    public float Trip_distance { get; set; }\n",
+        "\n",
+        "    [LoadColumn(5)]\n",
+        "    [ColumnName(@\"payment_type\")]\n",
+        "    public string Payment_type { get; set; }\n",
+        "\n",
+        "    [LoadColumn(6)]\n",
+        "    [ColumnName(@\"fare_amount\")]\n",
+        "    public float Fare_amount { get; set; }\n",
+        "\n",
+        "}"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        },
+        "vscode": {
+          "languageId": "dotnet-interactive.csharp"
+        }
+      },
+      "outputs": [
+        {
+          "data": {
+            "text/html": [
+              "<table><thead><tr><th>CanShuffle</th><th>Schema</th></tr></thead><tbody><tr><td><div class=\"dni-plaintext\">False</div></td><td><div class=\"dni-plaintext\">[ { vendor_id: String: Name: vendor_id, Index: 0, IsHidden: False, Type: { String: RawType: System.ReadOnlyMemory&lt;System.Char&gt; }, Annotations: { : Schema: [  ] } }, { rate_code: Single: Name: rate_code, Index: 1, IsHidden: False, Type: { Single: RawType: System.Single }, Annotations: { : Schema: [  ] } }, { passenger_count: Single: Name: passenger_count, Index: 2, IsHidden: False, Type: { Single: RawType: System.Single }, Annotations: { : Schema: [  ] } }, { trip_time_in_secs: Single: Name: trip_time_in_secs, Index: 3, IsHidden: False, Type: { Single: RawType: System.Single }, Annotations: { : Schema: [  ] } }, { trip_distance: Single: Name: trip_distance, Index: 4, IsHidden: False, Type: { Single: RawType: System.Single }, Annotations: { : Schema: [  ] } }, { payment_type: String: Name: payment_type, Index: 5, IsHidden: False, Type: { String: RawType: System.ReadOnlyMemory&lt;System.Char&gt; }, Annotations: { : Schema: [  ] } }, { fare_amount: Single: Name: fare_amount, Index: 6, IsHidden: False, Type: { Single: RawType: System.Single }, Annotations: { : Schema: [  ] } } ]</div></td></tr></tbody></table>"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "//Create MLContext\n",
+        "MLContext mlContext = new MLContext();\n",
+        "\n",
+        "// Create TextLoader based on the Model Input type. \n",
+        "TextLoader textLoader = mlContext.Data.CreateTextLoader<ModelInput>(separatorChar: ',', hasHeader: true);\n",
+        "\n",
+        "// Load the data into an IDataView. Load() method can support multiple files. \n",
+        "// Files must they have the same separator character, header, column names, etc. \n",
+        "IDataView data = textLoader.Load(\"data/taxi-fare.csv\");\n",
+        "\n",
+        "data.Preview(1); "
```

Comment:
I would take the code comments and create a md cell above with them. Something like:

> Create a `TextLoader` based on the `ModelInput` type and use the `Load` method to load your data.